<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QMainWindow" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>256</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="displayOrders">
        <property name="text">
         <string>Display orders</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addOrder">
        <property name="text">
         <string>Add an order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteOrder">
        <property name="text">
         <string>Delete an order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showProducts">
        <property name="text">
         <string>Show me my products</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSeller_Panel_Menu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuSeller_Panel_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
